src/lib389/lib389/tools.py

```diff
@@ -28,6 +29,7 @@
 import shlex
 # from .nss_ssl import nss_create_new_database
 from threading import Timer
+from lib389 import DirSrv
```

Comment:
CI fails to build here:

```
ImportError: cannot import name 'DirSrv' from partially initialized module 'lib389' (most likely due to a circular import) (/__w/389-ds-base/389-ds-base/rpmbuild/BUILD/389-ds-base-2.1.0.202203161041git81aaf6a/src/lib389/lib389/__init__.py)
```

Reply:
yes, this is due to internal very tight inter dependencies, fixed via deferred import.